src/service/repo/anonymousMessageRepo.ts

```diff
@@ -93,9 +97,13 @@ async function getAllAnonymousChannels() {
   }
 }
 
-async function getAllAnonymousPostNotifications() {
+async function getAllAnonymousPostNotifications(timeStamp: string | undefined) {
+  const url = timeStamp
+    ? `${baseUrl.getAllAnonymousPostNotifications}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllAnonymousPostNotifications;
+
   try {
-    const response = await anonymousApi.get(baseUrl.getAllAnonymousPostNotifications);
+    const response = await anonymousApi.get(url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this change introduces a conditional URL construction based on whether a `timeStamp` is provided or not for the `getAllAnonymousPostNotifications` function. Please ensure that the server-side API supports this query parameter and handles it correctly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->